aws-rds-dbproxyendpoint/pom.xml

```diff
@@ -61,11 +61,11 @@
             <version>2.26.0</version>
             <scope>test</scope>
         </dependency>
-
+        <!-- https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-rds -->
```

Comment:
why do we need to add this?

Reply:
it's just a hint where to look up for versions available. We had this added for proxy targets and missed for proxies and endpoints.